geomesa-core/src/test/scala/org/locationtech/geomesa/core/process/temporaldensity/TemporalDensityProcessTest.scala

```diff
@@ -0,0 +1,339 @@
+/*
+* Copyright 2014 Commonwealth Computer Research, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+package org.locationtech.geomesa.core.process.temporaldensity
+
+import com.vividsolutions.jts.geom.{Envelope, Point}
+import org.apache.accumulo.core.client.mock.MockInstance
+import org.apache.accumulo.core.client.security.tokens.PasswordToken
+import org.apache.hadoop.io.Text
+import org.geotools.data.simple.SimpleFeatureStore
+import org.geotools.data.{DataStore, DataUtilities, Query}
+import org.geotools.factory.Hints
+import org.geotools.filter.text.ecql.ECQL
+import org.geotools.filter.visitor.ExtractBoundsFilterVisitor
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.core.data._
+import org.locationtech.geomesa.core.index.{Constants, QueryHints}
+import org.locationtech.geomesa.core.iterators.TemporalDensityIterator.{decodeTimeSeries, TIME_SERIES, jsonToTimeSeries}
+import org.locationtech.geomesa.core.process.temporalDensity.TemporalDensityProcess
+import org.locationtech.geomesa.feature.AvroSimpleFeatureFactory
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.opengis.feature.simple.SimpleFeatureType
+import org.opengis.feature.simple.SimpleFeature
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+import scala.collection.JavaConversions._
+
+import org.joda.time.{DateTime, DateTimeZone}
+
+
+@RunWith(classOf[JUnitRunner])
+class TemporalDensityProcessTest extends Specification {
+
+  sequential
+
+  import org.locationtech.geomesa.utils.geotools.Conversions._
+
+  private val tableName = "tableTestTDIP"
+  private val sftName = "sftTestTDIP"
+
+  def createDataStore(sft: SimpleFeatureType, i: Int = 0): DataStore = {
+    val mockInstance = new MockInstance("dummy" + i)
+    val c = mockInstance.getConnector("user", new PasswordToken("pass".getBytes))
+    c.tableOperations.create(tableName)
```

Comment:
I copied this code from DensityIteratorTest.scala (and also TemporalDensityIteratorTest.scala) and am honestly not sure exactly what it is doing. Should I remove it from both of these locations as well as here?